File: c/chapter 8/largest.cpp
//largest number from matrix
#include<stdio.h>
#include<conio.h>
int main()
{
 int a[5][5];


int i,j,big;
printf("enter the 5*5 matrix");
for(i=0 ; i<5; i++)
{
for(j=0 ; j<5 ; j++)
{
printf("\nvalue at %d row %d column\n ",i+1,j+1);
scanf("%d", &a[i][j] );
}
}
big=a[0][0];
printf("The 5*5 matrix is:\n\n");
 for(i=0;i<5;i++)
  {
   for(j=0;j<5;j++)
   {
    printf("%d\t",a[i][j]);
    if(a[i][j]>big)
     {
      big=a[i][j];
     }
    }
 printf("\n");
}
printf("\nThe largest no. in this matrix is: %d",big);
getch();
}  

File: java/largest.java

import java.util.Scanner;
public class largest{
public static void main(String args[])
{
  int[][] a=new int[5][5];
  int b,i,j;
System.out.println("enter a 5*5 array\n");
Scanner in = new Scanner(System.in);
for(i=0;i<5;i++)
    {
for(j=0;j<5;j++)
{    
a[i][j] = in.nextInt();
}
}
b=a[0][0];

 for(i=0;i<5;i++)
  {
   for(j=0;j<5;j++)
   {
System.out.println(a[i][j]);
    if(a[i][j]>b)
     {
      b=a[i][j];
     }
    }
System.out.println("\n");
    }

System.out.println("\nThe largest no. in this matrix is:"+b);
}  
}

 

 
